functions/src/Services/Elections.ts

```diff
@@ -11,4 +13,18 @@ export const openElections = () => {
 		})
 		.then(() => Promise.resolve())
 		.catch(error => Promise.reject(error));
+};
+
+export const addPosition = (position: Position) =>{
+	const electionsCollection = getElectionsCollection();
+	const appendPosition = firestore.FieldValue.arrayUnion({ ...position });
+
+	electionsCollection.get()
+		.then((QuerySnapshot) => {
+			(QuerySnapshot.docs[0]).ref.update({ positions: appendPosition })
```

Comment:
also you should do a check to make sure that applications are open in order to add a position

Reply:
ok, I did not think of that, I'll change it